ios/Classes/AblyFlutter.m

```diff
@@ -29,6 +29,10 @@ -(void) setChannel:(FlutterMethodChannel *const)channel  {
     _channel = channel;
 }
 
+-(void) getNextHandle {
+    return @(_nextHandle++);
+}
```

Comment:
Two things here:
1. You're trying to return value from the `void` returning function
2. I'm not sure if `_nextHandle++` is what you want because `++` as a suffix will increment `_nextHandle` "later". I mean the returned  `_nextHandle++` value will be the same as `_nextHandle` (without `++`). If you want to increment this value in place and return incremented value you should use `++ _nextHandle`

Reply:
For #1, it's in draft and I am refactoring without building in some commits, and I've fixed that return value.

For #2, yup, that is exactly what it does. I know it's bad, but this is the existing logic of the class. I am merely refactoring it so that this type of logic is more obvious.,